be/src/common/config.h

```diff
@@ -424,8 +378,7 @@ CONF_Bool(enable_quadratic_probing, "false");
 CONF_String(pprof_profile_dir, "${DORIS_HOME}/log");
 
 // for partition
-// CONF_Bool(enable_partitioned_hash_join, "false")
-CONF_Bool(enable_partitioned_aggregation, "true");
+CONF_mBool(enable_partitioned_aggregation, "true");
```

Comment:
There is a memory leak risk, `enable_partitioned_aggregation` is true at first and then changed to false, memory allocated has no chance to free.